### DISCUSSION TITLE
Ran Performance test on my production app home page and I think I got banned

### DISCUSSION THREAD

**Discussion by zestsystem** [vercel/vercel#6134] [Help] [answered]
Hello

I was using k6 and following a tutorial on performance testing.  I got a 429 error and after that I could not access my app. I tried to check my vercel dashboard but I couldn't even go to vercel.com. So here I am. 

I didn't get any email saying I was banned but my friend was able to access the app no problem. And i was able to reach vercel.com  on cellular data.  So I think it's reasonable to assume that my ip is blacklisted... 

I will stick to just testing the routes of my express server. I apologize for being reckless and if I can get unbanned I will never do this again. 

Thank you for listening and I am very sorry.

**Comment by styfle** [comment, marked as answer]
Hi @mikeyim337 

When you see [status code 429](https://vercel.com/docs/api#api-basics/rate-limits), that means you exceed a rate limit and you'll need to wait until a certain period of time passes before trying again.

Hobby accounts can't perform Load Testing because its against our [Fair Use Policy](https://vercel.com/docs/platform/fair-use-policy).

Paying customers can create a [Support Request](https://vercel.com/support/request) to discuss load testing if that is a requirement.

**Reply by zestsystem** [reply]
Thank you very much for the response. I assume that for a high traffic web app I would need to upgrade from the hobby account. What would you say are the soft limit of concurrent users and requests per second of the hobby account before I need to upgrade?

**Reply by styfle** [reply]
You can find that in the Fair Use Policy above
